src/sfizz/SynthMessaging.cpp

```diff
@@ -165,7 +165,8 @@ void sfz::Synth::dispatchMessage(Client& client, int delay, const char* path, co
 
         MATCH("/region&/count", "") {
             GET_REGION_OR_BREAK(indices[0])
-            client.receive<'h'>(delay, path, region.sampleCount);
+            if (region.sampleCount)
+                client.receive<'h'>(delay, path, *region.sampleCount);
```

Comment:
can we still answer the request, without argument?

Reply:
Added !